.github/semantic.yml

```diff
@@ -0,0 +1 @@
+titleOnly: true
```

Comment:
no new line

Reply:
So if there will be anything in commit message body it will fail ?

Reply:
Added